test/cases/ast/_Float16.c

```diff
@@ -30,7 +30,7 @@ fn_def: 'fn (x: int, ...) void'
     builtin_call_expr: 'void'
      name: __builtin_va_start
      args:
-      implicit_cast: (array_to_pointer) 'va_list': '*d[1]struct __va_list_tag'
+      implicit_cast: (array_to_pointer) 'va_list': '*d[1]struct __va_list_tag' (value: )
```

Comment:
Would be nice if pointer values printed the offset (if non-zero).

Reply:
I meant to ask about this - should I just print the offset (without the name of the variable)?

Reply:
Printing the `NodeIndex` probably won't be too useful so I guess just the offset is good for now. Maybe add something to distinguish it from just ints.

Reply:
What if we change the return type of `Value.print` from `@TypeOf(w).Error!void` to something like `@TypeOf(w).Error!?NestedPrint` where `NestedPrint` is
```zig
const NestedPrint = union(enum) {
    pointer: struct {
        node: u32,
        offset: Value,
    },
};
```